package.json

```diff
@@ -20,10 +20,11 @@
     "bluebird": "^3.4.6",
     "debuglog": "^1.0.0",
     "disturbed": "^1.0.3",
+    "ioredis": "^2.4.0",
     "lodash": "^4.16.6",
     "mocha": "^3.1.2",
     "node-uuid": "^1.4.7",
-    "redis": "^2.6.3",
+    "redlock": "nextorigin/node-redlock",
```

Comment:
I don't think Bull should have this dependency. Please try to upstream your fix into node-redlock.